### DISCUSSION TITLE
Name Ideas

### DISCUSSION THREAD

**Discussion by deltasolidsnake** [WizardMaster38/Infernum-Ad-Infinitum-To-Hell-and-Beyond#8] [Ideas]
Ad Infernum Ad Infinitum (First "ad" optional)   (Means "To Hell and beyond" in Latin)

**Comment by WizardMaster38** [comment]
Nice, I like it! Maybe we could base the story off of Latin mythology?

**Reply by deltasolidsnake** [reply]
That could work! though I might have to read up on my mythology lol

**Reply by deltasolidsnake** [reply]
Or possibly the world is over run by demons and they have to fight to close the portal or somethin. (Literally DOOM)

**Reply by WizardMaster38** [reply]
I feel as if copying DOOM wouldn't help.

**Reply by WizardMaster38** [reply]
Actually, now that I think about it, have people done DOOM style things in this format?

**Reply by deltasolidsnake** [reply]
damn...

**Reply by deltasolidsnake** [reply]
Not that I know of.

**Reply by WizardMaster38** [reply]
We could do something like that.

**Reply by deltasolidsnake** [reply]
AYE
